.github/workflows/test-env-deploy.yml

```
@@ -68,11 +68,21 @@ jobs:
           pattern: (http|https)://[a-zA-Z0-9.-]+/?
         run: |
           echo "::set-output name=custom_marketplace_url::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
+      - name: Get APPS_MARKETPLACE_API_URI
+        id: apps_marketplace_api_uri
+        # Search for APPS_MARKETPLACE_API_URI in PR description
+        env:
+          pull_request_body: ${{ github.event.pull_request.body }}
+          prefix: APPS_MARKETPLACE_API_URI=
+          pattern: (http|https)://[a-zA-Z0-9.-].+/?
```

That `.` is dangerous as it allows anything. Did you mean:
```suggestion
          pattern: (http|https)://[a-zA-Z0-9.-][\w-/]+/?
```

---

I mean any URI also with subpath e.g. `https://marketplace-gray.vercel.app/api/v2/saleor-apps`

---

In that case the regex I suggest above should be enough

---

It doesn't work, what about `(http|https)://[a-zA-Z0-9.-]+[\w-/]+/?`?

---

Looks good. Should be (dash at the end):
```
(http|https)://[a-zA-Z0-9.-]+[\w/-]+/?
```